hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/config/HxInsightClientConfig.java

```diff
@@ -41,9 +41,9 @@ public final class HxInsightClientConfig
 
     public HxInsightClientConfig(@NotBlank String baseUrl)
     {
-        this.agentUrl = baseUrl + "/agents";
-        this.questionUrl = baseUrl + "/submit-question";
-        this.answerUrl = baseUrl + "/questions/%s/answer";
+        this.agentUrl = baseUrl + "/integrations/agents";
```

Comment:
Do we need to hard-code the `/integrations` part? Couldn't we pass it within `baseUrl`? 🤔

Reply:
Answer API path hasn't got `/integrations` prefix...

Reply:
I can see Jakub informed that "Polling answer will be added soon `GET /integrations/questions/{questionId}/answer?userId={userId}`"

Reply:
Sorry - I meant Feedback API... But that one isn't in Community Live scope and we can assume it will be changed.

Reply:
For me - it's okay to add `/integrations` as a part configurable `baseUrl`